### DISCUSSION TITLE
Men and women seperately or together when training style?

### DISCUSSION THREAD

**Discussion by keillnath** [TheLastBen/fast-stable-diffusion#1511] [Q&A]
When training for style do you train images of men and women separately or together?

**Comment by meunumerotim333** [comment]
I prefer separately because sometimes it Mix something.. and I don't know why or maybe I'm doing something wrong. But after I start to have some faces Mixed.. I've started to train faces just one by one.. and maybe you could use IMG2IMG or Inpaint to Mix them.

**Reply by ProgGrog** [reply]
I'm just curious.  Are you training on 1.5 or 2.1?  I had gotten good results in november on mixed training 1.5, but I haven't been able to replicate.  I'm working at 2.1 now and I was thinking to try separate and captioned.
